### DISCUSSION TITLE
Firmware needed

### DISCUSSION THREAD

**Discussion by bobrien-ssa** [mriscoc/Ender3V2S1#1210] [General]
I have an Ender 3 S1 (F4) with an upgraded heatblock (300c). Mine has the TJC display. Before the new heatblock, I was running the F4-UBL-TJC firmware. I now need the F4-UBL-T13 firmware. When I loaded this, the numbers on the auto mesh screen are missing again. Any way to get a F4-UBL-T13-TJC firmware? Is there any way to build the firmwares myself?

**Comment by bobrien-ssa** [comment]
I did try to build the firmare myself, but when using auto build marling I get this error:
region `FLASH' overflowed by 8580 bytes

**Reply by bagellord** [reply]
I am also getting that error. Looking for a solution myself, have you had any luck?

**Reply by Red369Ender3S1** [reply]
Any word on this has anyone of you found or made one that works I'm to dumb to build my own, been trying to get the Giadej compilation to flash to the display, no matter how I try I can't get it to I do have the tjc dislplay and upgraded the hot end to the pro sprite can anyone helps?
